### PR TITLE
Topic/clean main

### PR DESCRIPTION
**Pull request by NahuelVilla**
Hi, This pull request is for both, main and devel.

I am removing the examples that should not be released yet, I fixed the plots of the released example and I improved the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,57 @@
...
 
-This code is mainly based on python, the c++ is a work in progress which does not work yet.
+This repository is mainly based on python, the c++ part is a work in progress which does not work yet.
+
+
+## Dependancies
+
+`mpc_interface` depends on standard python libraries and `qpsolvers` + `osqp` for the examples.
+
+### Standard python packages
```
Comment (nim65s) [inline README.md:16]: Those are not "standard". By "standard", we mean packages from the Python Standard Library: https://docs.python.org/3/library/index.html
I think you should merge those 2 sections

Comment (NahuelVilla) [reply README.md:16]: perfect, I am pushing in 5 minuts

**Comment by nim65s** [comment]
All PR are for devel. If you want this to be included in main, let's first merge it on devel, and after that I'll make a release which will merge devel into main.

**Comment by nim65s** [comment]
Also, CI is not happy:

```
python/use_examples/simple_functional_example/biped_mpc_loop.py:13:1: E402 module level import not at top of file
python/use_examples/simple_functional_example/biped_mpc_loop.py:14:1: E402 module level import not at top of file
python/use_examples/simple_functional_example/biped_mpc_loop.py:16:1: E402 module level import not at top of file
python/use_examples/simple_functional_example/biped_mpc_loop.py:17:1: E402 module level import not at top of file

**Pull request merged**